docs/agave/archive/beginners-guides/_managing-data.rst

```diff
@@ -16,7 +16,7 @@ Directory listing
 
 .. code-block:: plaintext
 
-   files-list -v -S data.agaveapi.co $API_USERNAME
+   files-list -v files-list -v -S data.agaveapi.co $API_USERNAME
```

Comment:
Is this written correctly? files-list -v duplicated?

Reply:
yes, I removed the duplicate